financialaid/factories.py

```diff
@@ -0,0 +1,63 @@
+"""
+Factories for financialaid tests
+"""
+import datetime
+from factory import SubFactory
+
+from factory.django import DjangoModelFactory
+from factory.fuzzy import (
+    FuzzyAttribute,
+    FuzzyInteger,
+    FuzzyText,
+    FuzzyChoice, FuzzyFloat, FuzzyDateTime)
```

Comment:
I would import `fuzzy` and not the single imports, but if you want to keep in this way, please one import per line